public/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8">
-    <link rel="shortcut icon" href="../src/images/abstergo.jpg">
+    <link rel="shortcut icon" href="../src/components/atoms/abstergo.jpg">
     <!--<link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">-->
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <meta name="theme-color" content="#000000">
```

Comment:
(I leave comment here because I can't comment rows below)

You use PureCSS from CDN (`<link rel="stylesheet" href="https://unpkg.com/purecss@1.0.0/build/pure-min.css" integrity="sha384-nn4HPE8lTHyVtfCBi5yW9d20FjT8BJwUXyWZT9InLYax14RDjBj46LmSztkmNP9w" crossorigin="anonymous">`). Also you have installed PureCSS through Yarn, but you don't use this package. Try to get rid of from PureCSS from CDN and leave only PureCSS as Yarn package.

You can use PureCSS as package by adding `import 'purecss';` to `index.js`. (You can remove `index.css` import, because `index.css` contains styles for `body` and this styles can conflict with PureCSS)